noxfile.py

```diff
@@ -39,15 +41,17 @@ def run_readable(session, mode="fmt"):
 
 @nox.session(name="format", python=PYTHON_DEFAULT_VERSION)
 def format_(session):
-    session.run("pip", "install", "-e", ".[dev]")
+    session.run_always("pip", "install", "pdm")
```

Comment:
What I don't like about it: now we have to maintain (synchronize manually across all dev machines) not only `nox` but also `pdm`.
What I like about it: it will speed up the CI and `pdm` was made to work like that.

So I made the change, and lets see how it goes.

Reply:
What I don't like about it: now we have to maintain (synchronize manually across all dev machines) not only `nox` but also `pdm`.
What I like about it: it will speed up the CI and `pdm` was made to work like that.

So I made the change, and lets see how it goes.

Reply:
Wait, why we need to synchronize nox? I've never heard about something like that.

Reply:
I mean synchronizing the version of nox used

Reply:
We never discussed this and I don't think we really care